velox/connectors/hive/tests/HivePartitionUtilTest.cpp

```diff
@@ -117,3 +117,47 @@ TEST_F(HivePartitionUtilTest, partitionName) {
         "Unsupported partition type: MAP");
   }
 }
+
+TEST_F(HivePartitionUtilTest, partitionNameForNull) {
+  std::vector<std::string> partitionColumnNames{
+      "flat_bool_col",
+      "flat_tinyint_col",
+      "flat_smallint_col",
+      "flat_int_col",
+      "flat_bigint_col",
+      "dict_string_col",
+      "const_date_col"};
+
+  RowVectorPtr input = makeRowVector(
+      partitionColumnNames,
+      {makeNullableFlatVector<bool>(
+           std::vector<std::optional<bool>>{std::nullopt}),
```

Comment:
Can you drop 'std::vector<std::optional<bool>>'?

Reply:
sure, removed